### ISSUE TITLE
Integrations should have the information about what to manually test and how

### ISSUE THREAD

**Issue by ph** [elastic/package-spec#126]
The system integration is an important and special integration. We have dedicated resources that manually test the Elastic Agent and at the same time the system integration because it's so critical.  The system integration collect a lot of stuff and the behavior is different depending on the deployment machine.

Looking at issues created https://github.com/elastic/integrations/issues/570 and https://github.com/elastic/integrations/issues/551#issuecomment-764664670

I think we should find a way to to understand what actually need to be manually tested and how it can be manually tested, I want to make sure the team know the following:

**Data stream X should work on Windows but you need to have this configured or this need to be run to produce data.**

**Data Stream Y can be ignored because it's too complicated.**

If we don't do this, this will cause confusion from both the integration team and the testing team.

@EricDavisX @masci @amolnater-qasource @dikshachauhan-qasource

**Comment by ph** [comment]
Yes, +1 on this. what I like is the package will always be the source of truth.

@masci what is the best way to drive this with the integration team?

**Comment by fearful-symmetry** [comment]
Testing of any kind of the system package is sort of a nightmare, because we have so many OS considerations at play. We also need a compatibility matrix, since certain data streams require systemd, certain systemd configurations, certain kernels, etc. And that's just linux.
